src/main/java/org/gridsuite/modification/server/dto/TabularModificationInfos.java

```diff
@@ -35,11 +37,15 @@
 @JsonTypeName("TABULAR_MODIFICATION")
 @ModificationErrorTypeName("TABULAR_MODIFICATION_ERROR")
 public class TabularModificationInfos extends ModificationInfos {
-    @Schema(description = "Modification type")
+
+    public static final String TABULAR_EQUIPMENT_TYPE = "tabularEquipmentType";
+
+    @Schema(description = "equipment type")
     @NonNull
-    private String modificationType;
+    private IdentifiableType equipmentType;
```

Comment:
I think it's not enough. We can have here types like GENERATOR_MODIFICATION, GENERATOR_CREATION or EQUIPMENT_DELETION

Reply:
Using ModificationType enum would be more appropriate

Reply:
No `ModificationType` represente all modification types of  network
We need only injection modificatio/creation/deletion

Reply:
operationType { MODIFICATION, CREATION, DELETION } ?

Reply:
You want to use 2 fields : IdentifiableType equipmentType +  OperationType operationType ?

Reply:
Last decision: we use ModificationType enum. No data migration. Same Front-end.